libcrux-ml-kem/src/vector/portable/arithmetic.rs

```diff
@@ -42,7 +45,8 @@ pub fn add(mut lhs: PortableVector, rhs: &PortableVector) -> PortableVector {
 #[inline(always)]
 pub fn sub(mut lhs: PortableVector, rhs: &PortableVector) -> PortableVector {
     for i in 0..FIELD_ELEMENTS_IN_VECTOR {
-        lhs.elements[i] -= rhs.elements[i];
+        hax_lib::debug_assert!(lhs.elements[i] - rhs.elements[i] <= i16::MAX && lhs.elements[i] - rhs.elements[i] >= i16::MIN);
```

Comment:
Since https://github.com/hacspec/hax/pull/794 you can use Rust's `assert`. This would need a config guard like the import. Otherwise this doesn't compile
```
  --> libcrux-ml-kem/src/vector/portable/arithmetic.rs:48:9
   |
48 | ...   hax_lib::debug_assert!(lhs.elements[i] - rhs.elements[i] <= i16::MAX && lhs.element...
   |       ^^^^^^^ use of undeclared crate or module `hax_lib`
```